### DISCUSSION TITLE
daisyUI Organization And Official GitHub Account for daisyUI

### DISCUSSION THREAD

**Discussion by barziahmed** [saadeghi/daisyui#2829] [Ideas / feature requests]
Hello, 

DaisyUI is great, and I believe now it is time to move to the next stage, which is putting daisyUI in its own GitHub account and username, so the repository hopefully becomes something like daisyui/daiysui. This is to show that daisyUI is an organization and important for promoting and marketing it for companies and other entities which rely on trustworthy packages and plugins to put in their production.

**Comment by saadeghi** [comment]
Thanks for the suggestion.  
I think moving daisyUI to a new URL solely for the sake of a different username doesn't inherently enhance its credibility or utility.   
Companies regularly rely on numerous Node packages, most of which reside in personal GitHub accounts without compromising trust or functionality.

**Comment by ruchernchong** [comment]
@saadeghi any update on this decision yet?

**Reply by saadeghi** [reply]
If you think there are any benefits in this movement, I'm listening.
